matchmaker/views/external_api.py

```diff
@@ -70,9 +72,12 @@ def mme_match_proxy(request, originating_node_name):
 
     try:
         query_patient_data = json.loads(request.body)
+    except Exception:
```

Comment:
not all exceptions are JSON errors, you need to be explicitly only catching JSON errors here

Reply:
Updated. But since Python 2 doesn't have JSONDecodeError, special handling is added.